### PR TITLE
Migrate published samples

### PR DESCRIPTION
**Pull request by texasmichelle**
Migrate from tmp-generated-samples branch 615c08e
Remove boilerplate
Update copyright date
Blacken
Remove unused imports
Shorten docstrings
Remove CLI
Set defaults in function definition

### CODE REVIEW

File: translate/cloud-client/translate_v3_translate_text.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3_translate_text]
+from google.cloud import translate
+
+
+def sample_translate_text(
+    text="Hello, world!", target_language="fr", project_id="[Google Cloud Project ID]"
```
Comment (nnegrey) [inline translate/cloud-client/translate_v3_translate_text.py]: Thoughts about inlining `text` and `target_language`

below would be:
```
response = client.translate_text(
        parent=parent,
        contents=["Hello, world!"],
        mime_type="text/plain",  # mime types: text/plain, text/html
        source_language_code="en-US",
        target_language_code="fr",
    )
```

File: translate/cloud-client/translate_v3_translate_text.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    client = translate.TranslationServiceClient()
+
+    contents = [text]
+    parent = client.location_path(project_id, "global")
+
+    response = client.translate_text(
+        parent=parent,
+        contents=contents,
+        mime_type="text/plain",  # mime types: text/plain, text/html
```
Comment (nnegrey) [inline translate/cloud-client/translate_v3_translate_text.py]: Nit: might be useful to add a link above on like line 28 to this page: https://cloud.google.com/translate/docs/supported-formats

File: translate/cloud-client/translate_v3_translate_text.py (hunk 1 above)
Comment (texasmichelle) [reply translate/cloud-client/translate_v3_translate_text.py]: Yes, that's simpler.

File: translate/cloud-client/translate_v3_translate_text.py (hunk 2 above)
Comment (texasmichelle) [reply translate/cloud-client/translate_v3_translate_text.py]: Done

**Pull request merged**